modules/Bio/EnsEMBL/Analysis/Hive/Config/Genome_annotation_conf.pm

```diff
@@ -823,6 +827,18 @@ sub default_options {
     'dump_features_exe'                 => catfile($self->o('enscode_root_dir'),'ensembl-compara/scripts/dumps/DumpMultiAlign.pl'),
 
 
+    #######################
+    # RNASamba CPC2 Stuff
+    #######################
+
+    'rna_samba_weights' => '/homes/jma/coding_gene_testing/full_length_weights.hdf5',
```

Comment:
If this path could be turned into a variable or more central location it would be great.

Reply:
Moved to a temp location. Will arrange a proper central install ahead of integration to dev/hive_master